Makefile

```diff
@@ -60,9 +60,95 @@ build-all:
 aro: check-release generate
 	go build -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro
 
+# Target to deploy the CosmosDB database and container
+.PHONY: deploy-database
+deploy-database:
+	@echo "Deploying CosmosDB database and container..."
+	@az deployment group create \
+		-g "$(RESOURCEGROUP)" \
+		-n "databases-development-$(USER)" \
+		--template-file pkg/deploy/assets/databases-development.json \
+		--parameters \
+			databaseAccountName="$(DATABASE_ACCOUNT_NAME)" \
+			databaseName="$(DATABASE_NAME)" \
+		1>/dev/null || echo "Deployment failed or already exists"
+
+# Target to login to ACR
+.PHONY: acr-login
+acr-login:
+	@echo "Logging into ACR..."
+	@echo "PULL_SECRET: $${PULL_SECRET}"
+	@acr_auth=$$(echo "$${PULL_SECRET}" | jq -r '.auths["arointsvc.azurecr.io"].auth' | base64 -d); \
+	echo "Decoded ACR Auth: $$acr_auth"; \
+	username=$$(echo "$$acr_auth" | cut -d: -f1); \
+	password=$$(echo "$$acr_auth" | cut -d: -f2); \
+	echo "ACR Username: $$username"; \
+	echo "ACR Password: $$password"; \
+	docker login arointsvc.azurecr.io --username $$username --password $$password
+
+# Target to run the local RP
 .PHONY: runlocal-rp
-runlocal-rp:
-	go run -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro rp
+runlocal-rp: deploy-database acr-login
```

Comment:
we should add `make ci-rp` as a precursor so we guarantee we get the docker image built, and all changes are there if we made any.

Reply:
Added the ci-rp